libsqlite3-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libsqlite3-sys"
-version = "0.17.1"
+version = "0.18.0"
```

Comment:
Why 0.18.0 and not 0.17.2?

Reply:
@thomcc From previous pull requests where the SQLite version was updated, it appears that the major version was increased.

Reply:
If needed I can always change it to 0.17.2.